src/stores/Checkout.js

```diff
@@ -472,6 +473,36 @@ class CheckoutStore {
     }
   });
 
+  EntitlementClaim = flow(function * ({entitlementSignature}) {
+    const decode = yield this.client.DecodeSignedMessageJSON({signedMessage: entitlementSignature});
+    decode?.message && rootStore.log("EntitlementClaim msg", decode.message);
+    const tenant_id = decode?.message?.tenant_id;
+
+    const tok = this.walletClient.AuthToken();
+    const url = UrlJoin(this.walletClient.client.authServiceURIs[0], "wlt", "act", tenant_id);
```

Comment:
Looks like this is still hardcoded to use a particular authd node. This whole request section can just be written:

```
      yield this.client.authClient.MakeAuthServiceRequest({
        path: UrlJoin("as", "wlt", "act", tenantId),
        method: "POST",
        body: params,
        headers: {
          Authorization: `Bearer ${this.rootStore.authToken}`
        }
      });
```